docs/docs/event-bus.md

```diff
@@ -29,6 +29,34 @@ Other subscribers are easy to add. See the code in
 [marathon/event/http](https://github.com/mesosphere/marathon/tree/master/src/main/scala/mesosphere/marathon/event/http)
 for guidance.
 
+## Subscription to Events via event stream
+
+This functionality is always enabled. Marathon implements
+[Server-sent events](https://en.wikipedia.org/wiki/Server-sent_events) standard.
+Events are published on `/v2/events` endpoint. Any SSE compatible client
+could be used to subscribe. Example subscription using `curl`:
+
+```bash
+$ curl -H "Accept: text/event-stream"  <MARATHON_HOST>:<MARATHON_PORT>/v2/events
+
+event: event_stream_attached
+data: {"remoteAddress":"127.0.0.1","eventType":"event_stream_attached","timestamp":"2017-02-18T19:12:00.102Z"}
+```
+
+### Filtering event stream
+
+From version [1.3.7](https://github.com/mesosphere/marathon/releases/tag/v1.3.7)
```

Comment:
Currently user can find available event types only in code. There is dedicated issue for this https://github.com/mesosphere/marathon/issues/4672